### PR TITLE
translation cloud_native_technology

### PR DESCRIPTION
**Pull request by sistella**

### CODE REVIEW

File: content/it/cloud_native_tech.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Il _cloud native stack_ è formato da molte categorie di tecnologie diverse tra loro; ciascuna si propone di risolvere un problema specifico. Consultando il [CNCF Cloud Native Landscape](https://landscape.cncf.io/), potrai renderti conto di quante aree differenti coinvolga. In linea generale, il _cloud native stack_ affronta un gruppo principale di problemi, ossia gli aspetti negativi dei modelli operativi IT tradizionali. Fra gli altri vi sono: la difficoltà di creare applicazioni scalabili, _fault-tolerant_ (tolleranti gli errori) e [_self-healing_](/self_healing) (auto riparanti), o anche l'utilizzo inefficiente delle risorse.
+
+## In che modo aiuta
+
+Sebbene ogni tecnologia risponda ad un bisogno specifico, collettivamente le tecnologie _cloud native_ consentono sistemi disaccoppiati che sono insieme _resilient_(affidabili), gestibili ed osservabili. Queste caratteristiche, combinate ad un'automazione stabile, consentono agli sviluppatori di implementare cambiamenti ad alto impatto frequentemente e con esito certo a fronte di uno sforzo minimo. In sostanza, gli aspetti peculiari dei sistemi _cloud native_ si ottengono più facilmente con l'utilizzo del _cloud native stack_.
```
Comment (annalisag-spark) [inline content/it/cloud_native_tech.md:17]: - (affidabili) > non possiamo usare resilienti? ha un altro significato in questo contesto?
- gli aspetti peculiari dei sistemi _cloud native_ si ottengono più facilmente con l'utilizzo del _cloud native stack_ > gli aspetti non si "ottengono" secondo me. magari si valorizzano? si potenziano?

File: content/it/cloud_native_tech.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Quale problema affronta 
+
+Il _cloud native stack_ è formato da molte categorie di tecnologie diverse tra loro; ciascuna si propone di risolvere un problema specifico. Consultando il [CNCF Cloud Native Landscape](https://landscape.cncf.io/), potrai renderti conto di quante aree differenti coinvolga. In linea generale, il _cloud native stack_ affronta un gruppo principale di problemi, ossia gli aspetti negativi dei modelli operativi IT tradizionali. Fra gli altri vi sono: la difficoltà di creare applicazioni scalabili, _fault-tolerant_ (tolleranti gli errori) e [_self-healing_](/self_healing) (auto riparanti), o anche l'utilizzo inefficiente delle risorse.
```
Comment (annalisag-spark) [inline content/it/cloud_native_tech.md:13]: tolleranti gli errori > tolleranti agli errori

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

File: content/it/cloud_native_tech.md (hunk 2 above)
Comment (sistella) [reply content/it/cloud_native_tech.md]: Fatto!

**Review by annalisag-spark — APPROVED** [review]

**Review by meryem-ldn — APPROVED** [review]

File: content/it/cloud_native_tech.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cloud Native Technology
```
Comment (jihoon-seo) [inline content/it/cloud_native_tech.md:2]: Please consider translate "Cloud Native Technology" into Italian, if needed!

Ref: "API Gateway" page in pt-br:
- https://github.com/cncf/glossary/pull/479/files
- https://deploy-preview-479--cncfglossary.netlify.app/pt-br/api_gateway/

image

File: content/it/cloud_native_tech.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+status: Completed
+category: Concept
```
Comment (jihoon-seo) [inline content/it/cloud_native_tech.md:4]: Please consider translate "Concept" into Italian, if needed!

Ref: "API Gateway" page in pt-br:
- https://github.com/cncf/glossary/pull/479/files
- https://deploy-preview-479--cncfglossary.netlify.app/pt-br/api_gateway/

image

**Review by jihoon-seo — COMMENTED** [review]
@sistella PTAL!

File: content/it/cloud_native_tech.md (hunk 3 above)
Comment (sistella) [reply content/it/cloud_native_tech.md:2]: Thanks @seokho-son , we agreed keeping this in English makes more sense in Italian!

File: content/it/cloud_native_tech.md (hunk 4 above)
Comment (sistella) [reply content/it/cloud_native_tech.md:4]: Thanks @seokho-son , that's a good point, we will implement the change!

**Pull request merged**